GameEngine/PlayerHand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameEngine
+{
+    public class PlayerHand
+    {
+        private static readonly Random Random = new Random();
+        public List<CardType> Cards { get; private set; }
```

Comment:
💡 I _think_ you can remove the setter here, since we're only assigning to it in the constructor(s).

Reply:
Yeah, I think this was left over when I was toying around with the idea of initializing Players with starting Hands. Removed!